Web.md

```diff
@@ -139,27 +139,32 @@ following conventions are adopted.
 A WebAssembly location is a reference to a particular instruction in the binary, and may be
 displayed by a browser or engine in similar contexts as JavaScript source locations.
 It has the following format:
-`${url}:wasm-function[${funcIndex}]:0x${pcOffset}`
+`${url}:wasm-function[${funcIndex}]:${pcOffset}`
 Where
-* `${url}` is the URL associated with the module (e.g. via a response object), if any.
-* `${funcIndex}` is an index the [function index space](https://github.com/WebAssembly/design/blob/master/Modules.md#function-index-space).
-* `${pcOffset}` is the offset in the module binary of the first byte of the instruction, printed in hexadecimal with lower-case digits.
+* `${url}` is the URL associated with the module (e.g. via a response
+  object), or other module identifier (see notes).
+* `${funcIndex}` is an index the [function index space](Modules.md#function-index-space).
+* `${pcOffset}` is the offset in the module binary of the first byte
+  of the instruction, printed in hexadecimal with lower-case digits,
+  with a leading `0x` prefix.
 
 Notes:
 * The URL field may be interpreted differently depending on the context. For
 example offline tools may use a file name; or when the ArrayBuffer-based
 `WebAssembly.instantiate` API is used in a browser, it may display the
-location of the API call instead.
+location of the API call instead. It should not be empty however; a
+developer should be able to write their code such that modules from
```

Comment:
Are you suggesting that programmers should be able to rely on unambiguous location URLs? If so, I don't think that can work in general, e.g. for the aforementioned reason, but also because of URL ambiguities in general. I would rather drop that half-sentence.